src/app/collective-rewards/settings/builder/BuilderRewardsSettingsMetrics.tsx

```diff
@@ -0,0 +1,82 @@
+import { MetricsCard, MetricsCardRow, MetricsCardTitle } from '@/app/collective-rewards/rewards'
+import { withSpinner } from '@/components/LoadingSpinner/withLoadingSpinner'
+import { Typography } from '@/components/Typography'
+import { DateTime } from 'luxon'
+import { FC, useEffect } from 'react'
+import { Address } from 'viem'
+import { useAccount } from 'wagmi'
+import { useGetBackerRewardsForBuilder } from '../hooks'
+import { weiToPercentage } from '../utils'
+import { useBuilderSettingsContext } from './BuilderSettings'
+
+const getDateTimeRemaining = (cooldownEndTime: DateTime | undefined): string | undefined => {
+  if (!cooldownEndTime) {
+    return
+  }
+
+  const now = DateTime.now()
+  const remaining = cooldownEndTime.diff(now, ['days', 'hours', 'minutes'])
+  if (remaining.as('minutes') < 0) {
+    return
```

Comment:
should we specify something like "less than a minute"? or even 1 minute I guess it will be fine.

Reply:
I'd leave it for "nice to have"s as it's not in designs either.